docs/fx-core/config.md

```diff
@@ -15,4 +69,96 @@ botId|string|The AAD App client id generated during provision which is paired wi
 botPassword|string|The AAD App client secret generated during provision which is paired with botId to provide authentication between the bot project and the bot framework service.
 objectId|string|The AAD App object id generated during provision which is co-related with botId and botPassword.
 appServicePlan|string|The Azure App Service Plan name for the Azure Web App which is used for hosting the bot project on Azure.
-botChannelReg|string|The Azure Bot Channels Registration name for bot registration.
\ No newline at end of file
+botChannelReg|string|The Azure Bot Channels Registration name for bot registration.
+
+## Configuration schema for MS identity
+This section is to describe configuration items in `fx-resource-identity` section of `.fx/env.default.json`.
+
+Config Name | Config Type | Description
+------|------|------
+identity	|string	|Identity present name end user input/
+identityName|	string|	Full path resource name. '/subscriptions/${subscriptionId}/resourcegroups/${resourceGroup}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/${identity}'
+identityId|	string|	Client Id of the identity.
+
+## Configuration schema for frontend hosting
+
+This section is to describe configuration items in `fx-resource-frontend-hosting` section of `.fx/env.default.json`.
+
+Config Name | Config Type | Description
+------|------|------
+endpoint|	string|	Output. The endpoint of static website. https://domain.com
+domain	|string|	Output. The domain of static website. domain.com
+storageName|	string|	Output. The name of the Azure Storage Account where static website is hosted.
+
+## Configuration schema for Azure SQL
+
+This section is to describe configuration items in `fx-resource-azure-sql` section of `.fx/env.default.json`.
+
+Config Name | Config Type | Description
+------|------|------
+sqlEndpoint	|string|	The sql server endpoint.  '${sqlServer}.database.windows.net'
+databaseName|	string|	The created database name
+userName|	string|	account login for local debug
+password|	string|	account password for local debug
+
+## Configuration schema for Azure Function
```

Comment:
please change `Azure Function` to `Azure Functions`, thanks, that's the correct term.